### ISSUE TITLE
[Bug]: cannot publish documents that where created with pimcore 6.9

### ISSUE THREAD

**Issue by jheimbach** [pimcore/pimcore#11801]
### Expected behavior

I want to publish a document which was created before the update to pimcore 10.x (currently on 10.2.5)

Only happens if the document was never published.

### Actual behavior

I get the error `cannot access property starting with \0`.

The error is thrown here: https://github.com/pimcore/pimcore/blob/10.x/models/Element/Service.php#L1227
called from `Pimcore\Model\Document\PageSnippet::checkMissingRequiredEditable` 

### Steps to reproduce

(migration script was executed prior to pimcore 10 update)

I think it is because the version has a serialized Page Object which has still properties `module` `action` and `elements` but these where removed. 

I'm not sure how to fix it, because I have no idea how to remove properties from an unserialized object of a class that does not have these properties (or serialized without some very janky regex foo). If someone has an idea, i can create a PR for that.

useful grep command for finding the affected versions: `grep -rnw './var/versions/document' -e 'O:27:"Pimcore\\Model\\Document\\Page":36'`

**Comment by NiklasBr** [comment]
Had the same error here: https://github.com/pimcore/pimcore/issues/10661#issuecomment-979886344= but it was never fixed, issue was closed when only one of the problems were fixed.

**Comment by dvesh3** [comment]
@JHeimbach As this was fixed in 10.2.6 version https://github.com/pimcore/pimcore/pull/10920. could you please give it a try with updating Pimcore? thanks!

**Comment by NiklasBr** [comment]
@dvesh3 that PR fixed 
```
Twig\Error\RuntimeError:
Key "1" for array with keys "20" does not exist.
```
But not
```
cannot access property starting with \0
```

**Comment by dvesh3** [comment]
@NiklasBr please provide a PR to fix the issue, since I cannot reproduce it at my end. thanks!

**Comment by NiklasBr** [comment]
@dvesh3 Please see https://github.com/pimcore/pimcore/issues/10661#issuecomment-981922328= for more info

**Comment by jheimbach** [comment]
the suggested solution

```
try {
    $myValue = $reflectionProperty->getValue($object);
} catch (\Throwable $e) {
    return false;
}
```
works for me, will create a PR to fix it.

**Comment by dvesh3** [comment]
Fixed by #11806

**Comment by Cruiser13** [comment]
We actually experienced the exactly same bug on Pimcore v10.5.10 - is it possible that this has been reintroduced at some version @dvesh3 ?
Just updated from 6.9.6 to 10.5.10 and all never published documents are throwing this error:

> Status: 500 | Internal Server Error
URL: /admin/page/get-data-by-id
Message: Cannot access property starting with "\0"
Trace: 
in /serverpath/pimcore/vendor/pimcore/pimcore/lib/Tool/Serialize.php:105
#0 /serverpath/pimcore/vendor/pimcore/pimcore/lib/Tool/Serialize.php(87): Pimcore\Tool\Serialize::loopFilterCycles()
#1 /serverpath/pimcore/vendor/pimcore/pimcore/lib/Tool/Serialize.php(87): Pimcore\Tool\Serialize::loopFilterCycles()
#2 /serverpath/pimcore/vendor/pimcore/pimcore/lib/Tool/Serialize.php(75): Pimcore\Tool\Serialize::loopFilterCycles()
#3 /serverpath/pimcore/vendor/pimcore/pimcore/bundles/AdminBundle/Serializer/Normalizer/ReferenceLoopNormalizer.php(32): Pimcore\Tool\Serialize::removeReferenceLoops()
#4 /serverpath/pimcore/vendor/symfony/serializer/Serializer.php(161): Pimcore\Bundle\AdminBundle\Serializer\Normalizer\ReferenceLoopNormalizer->normalize()
#5 /serverpath/pimcore/vendor/symfony/serializer/Serializer.php(134): Symfony\Component\Serializer\Serializer->normalize()
#6 /serverpath/pimcore/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/AdminController.php(219): Symfony\Component\Serializer\Serializer->serialize()
#7 /serverpath/pimcore/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/AdminController.php(264): Pimcore\Bundle\AdminBundle\Controller\AdminController->encodeJson()
#8 /serverpath/pimcore/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/DocumentControllerBase.php(100): Pimcore\Bundle\AdminBundle\Controller\AdminController->adminJson()
#9 /serverpath/pimcore/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/PageController.php(103): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\DocumentControllerBase->preSendDataActions()
#10 /serverpath/pimcore/vendor/symfony/http-kernel/HttpKernel.php(163): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\PageController->getDataByIdAction()
#11 /serverpath/pimcore/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\Component\HttpKernel\HttpKernel->handleRaw()
#12 /serverpath/pimcore/vendor/symfony/http-kernel/Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle()
#13 /serverpath/pimcore/public/index.php(36): Symfony\Component\HttpKernel\Kernel->handle()
#14 {main}
